README.md

```diff
@@ -81,15 +81,15 @@ With it, it's possible to influence migration with ease to adjust the country's
 
 ![HighClassBanner](https://user-images.githubusercontent.com/32886642/210163119-4ba0270f-27b1-4a5f-8eef-1c59383f94e9.png)
 
-TGC's oficial music sub mod! Hours of carefully curated classic and folk music to enchance your Victoria II gameplay. Each music was carefully choosen, and assigned special playing conditions, from war, peace, country, culture, even war exhaustion and level of westernization!
+TGC's official music sub mod! Hours of carefully curated classic and folk music to enhance your Victoria II gameplay. Each music was carefully chosen, and assigned special playing conditions, from war, peace, country, culture, even war exhaustion and level of westernization!
```

Comment:
Suggestion;
TGC's official music submod! With hours of carefully curated classic and folk music, to enhance your Victoria II gameplay. The music was [...]